### ISSUE TITLE
ConfigurationSaver does not consider FreezeButs

### ISSUE THREAD

**Issue by Elscha** [SSEHUB/EASyProducer#39]
Given the followng example:

<pre><code>project PartialCompoundAssignment {

    version v0;
    enum BT {compile_time, run_time};
    annotate BT bindingtime = BT.compile_time to PartialCompoundAssignment;
    compound Dim {
        Integer width;
        assign (bindingtime = BT.run_time) to {
            Integer height;
        }
    }

    Dim dim1 = {width = 1, height = 1};
    Dim dim2 = {width = 2, height = 1};
    freeze {
        dim1;
        dim2;
    } but (var | var.bindingtime == BT.run_time)
}</code></pre>

After changing values and saving, the but block of the freeze block will be removed (see second last line).

<pre><code>project PartialCompoundAssignment {

    version v0;
    enum BT {compile_time, run_time};
    annotate BT bindingtime = BT.compile_time to PartialCompoundAssignment;
    compound Dim {
        Integer width;
        assign (bindingtime = BT.run_time) to {
            Integer height;
        }
    }

    Dim dim1 = {width = 1, height = 1};
    Dim dim2 = {width = 2, height = 1};
    freeze {
        dim1;
        dim2;
    } 
}</code></pre>

**Comment by eichelbe** [comment]
already done?

**Comment by Elscha** [comment]
No. ConfigurationSaver creates a nw freeze block considering all frozen elements correctly. However if elements where frozen in the same project, freeze buts are not correctly processed.
